examples/otlp-push-controller/README.md

```diff
@@ -0,0 +1,30 @@
+# Autometrics + OTLP push controller
+
+This example demonstrates how you can push autometrics via OTLP gRPC protocol to an OTEL-Collector or another compatible solution.
+
+## ⚠️ Warning
+
+At this step, it's absolutely required that the version of the opentelemetry crates used in the project matches the version used as a dependency in `autometrics-rs` crate (See [Issue 91](https://github.com/autometrics-dev/autometrics-rs/issues/91)).
+
+## Running the example
+
+### Start a basic OTEL-Collector
+
+You can use the `otel-collector-config.yaml` file to start an otel-collector container that listen on 0.0.0.0:4317 for incoming otlp-gRPC traffic, and export received metrics to standard output.
+
+```bash
+docker run -d --name otel-col \
```

Comment:
It might be better to run this in the interactive mode so that it's easy to kill it